apps/reviews/views.py

```diff
@@ -8,24 +8,24 @@
 class ProductReviewView(ModelViewSet):
     queryset = ProductReview.objects.all()
     serializer_class = ProductReviewSerializer
-    permission_classes = [AllowAny]
+    permission_classes = [IsAuthenticated]
 
-    http_method_names = [m for m in ModelViewSet.http_method_names if m not in ["put"]]
+    http_method_names = [
+        m for m in ModelViewSet.http_method_names if m not in ["put", "patch"]
+    ]
 
-    def get_permissions(self):
```

Comment:
Permission for list and read show still be opened... 
Everybody should be able to see the reviews of a product on the app 

But only authenticated users can create, update or delete their reviews
You can also check to make sure other users dont delete other people's review